tree/dataframe/inc/ROOT/RDF/RLoopManager.hxx

```diff
@@ -135,9 +137,9 @@ class RLoopManager : public RNodeBase {
    static unsigned int GetNextID();
 
 public:
-   RLoopManager(TTree *tree, const ColumnNames_t &defaultBranches);
+   RLoopManager(TTree *tree, const ColumnNames_t &defaultBranches, ULong64_t startEvt = 0ULL, ULong64_t endEvt = 0ULL);
```

Comment:
documentation should clarify the interplay between these .

Also, usually we spell things as explicitly as possible in user-visible function arguments, I'd suggest `startEntry` (and `fStartEntry` for the data-member) -- entry vs event is a pet peeve of mine, but e.g. with an `Explode` the rows woul not correspond to events anymore